.github/workflows/nmodl-ci.yml

```diff
@@ -143,6 +143,7 @@ jobs:
         env:
           CCACHE_BASEDIR: ${{runner.workspace}}/nmodl
           CCACHE_DIR: ${{runner.workspace}}/ccache
+          PYTHONMALLOC: malloc
```

Comment:
Changes to be discussed

Reply:
Seems OK to me.

Context: Catch2 does not set the full environment (https://github.com/catchorg/Catch2/issues/1810) when running test executables to discover the list of tests.

When LeakSanitizer is enabled, it flags leaks from Python if we do not set `PYTHONMALLOC=malloc`.